### ISSUE TITLE
Allow shaping of SDR BT1886 curve

### ISSUE THREAD

**Issue by rtguy** [arvehj/jvcprojectortools#16]
As you may know the JVC can have an issue with gamma droop, where the upper end sags. The JVC autocal corrects this. However when I try to use the BT1886 curve for SDR, the gamma droop is present.

Is it possible to add a feature / parameter that would enable me to "pull up" the top end to keep it at 2.37at 85, 90, and 95% so it does not droop? Also it would be nice if I could shape the lower end,,,  As you can see I have 5% perfect, but all the other levels are a little high,  This was as close to BT1886 I could get (according to CalMAN's BT1886 gamma curve calibration setting) by using brightness compensation (eb) of 0.0035 (the default value started the curve with 5% at a gamma too high of about 2.3). 

Below is a screen of what the gamma curve looked like as best as I could get it. Thank you!

![1886](https://cloud.githubusercontent.com/assets/2348809/24087785/a7bfdaec-0cf9-11e7-930e-48dd3ff2a42e.png)

**Comment by arvehj** [comment]
Did you measure a 2.2 curve? After autocal the 2.2 and standard curves should measure 2.2. If they don't I can add a parameter to set the output gamma, but it is probably better to re-run autocal.

If your 2.2. curve is correct, have you tried combining bbo with bbi / eb? If you set eb to 0 the gamma curve should be 2.4 at the top.

**Comment by rtguy** [comment]
Thanks. I'm going to rerun autocal. It may have been an issue with the positioning of the Spyder 5 meter. Too bad they don't support other meters like the i1D3.

**Comment by rtguy** [comment]
Hi Arve - Have you verified that the internal "baseline" gamma table that is set when you run an autocal is also applied to imported curves? Or is it possible that imported curves bypass whatever baseline the internal gamma tables have set?

I did get a pretty even gamma (slight droop to 2.1 at 95%) after rerunning autocal (I think my meter position wasn't set right the previous time).  However when I imported your BT1886 curve the droop was quite significant even tho it is not with the baseline 2.2 internal curve.

In the BT1886 curve shown I used an eb of 0.005 (I think, something around there, can't recall exactly).  I did not use bbo or bbi as I don't understand how I am suppose to use that and or combine that with eb.  What I can say is that with eb at the 0 default the gamma was way too high on the low end (5-20% were way above the yellow line, something in the 2.3-2.4 range.  When I set eb then the curve started lower and looks great as you can see, until it gets into the upper range.  Thank you!

![gamma-2-2](https://cloud.githubusercontent.com/assets/2348809/24308746/a39322ae-109f-11e7-8b87-d8135df56e65.png)

![gamma-bt1886](https://cloud.githubusercontent.com/assets/2348809/24308757/abe8da48-109f-11e7-9d97-f0b9bb23227f.png)

**Comment by arvehj** [comment]
Yes, the autocal correction applies to the custom gamma tables. I use super-white and soft clipping, so if there is a problem at the top end, I would not notice it though.

Do you know what the gamma numbers reported for each measurement mean? You cannot calculate a gamma from a single point, so these numbers have to be related to some other measurement. If it is relative to white, then a bad measurement for white would have a large effect on points that are close to white (and your white number does not seem to be calculated the same way as the other points).

The custom gamma curve might be a little brighter than the standard curve. If you want to correct for this, try lowering reference white so the output value shown on the graph does not exceed 1020.
